.gitignore

```diff
@@ -1 +1,2 @@
 .DS_Store
+.vscode/launch.json
```

Comment:
offroad: I added a launch config to debug rust and was told not to check it in, even though I thought it might be helpful to others.

Reply:
I think this should be kind what we did for example.yaml, where we provide examples but don't check in the one people use in case they want different settings. Adding .vscode/launch.example would be good I think.

Reply:
good idea! I've put my current launch in - feel free to amend / improve / fix the preLaunchTask ( that wasn't working due to `server` being in a subdir )